assignment-5/insertUser.js

```diff
@@ -0,0 +1,20 @@
+module.exports.insertUser = function(app, user) {
+  app.post('/insertUser', function(req, res) {
+    var isUserAlreadyPresent;
+    var receivedID = req.body.id
+    var isUserAlreadyPresent = user.findIndex((user,index)=> { 
```

Comment:
@JainVayam findIndex functions used multiple times. we can create the function in the separate file and call it.